third_party/cloud-provider-vsphere/pkg/common/config/config_yaml.go

```diff
@@ -205,7 +205,6 @@ func ReadRawConfigYAML(byConfig []byte) (*CommonConfigYAML, error) {
 	}
 
 	if err := yaml.Unmarshal(byConfig, &cfg); err != nil {
-		klog.Errorf("Unmarshal failed: %s", err)
```

Comment:
I do not think we need this logged, if INI passes we don't need to know yaml unmarshal failed and if it doesn't then we would have the same error message twice in the logs.